.github/workflows/break-pacman-upload-lease.yml

```diff
@@ -0,0 +1,40 @@
+name: break-pacman-upload-lease
+run-name: Break pacman upload lease
+
+on:
+  workflow_dispatch:
+
+jobs:
+  break-lease:
+    runs-on: windows-latest
+    steps:
+      - uses: git-for-windows/setup-git-for-windows-sdk@v1
+        with:
+          flavor: minimal
+          architecture: x86_64
```

Comment:
Do we even need the minimal SDK? As far as I can tell, [`pacman-helper.sh break_lock`](https://github.com/git-for-windows/build-extra/blob/HEAD/pacman-helper.sh) simply hands off to [`wingit-snapshost-helper.sh`](https://github.com/git-for-windows/build-extra/blob/HEAD/wingit-snapshot-helper.sh), which calls `curl`. GitHub workflows' `bash` steps use Git for Windows' Bash (!!!), which is distributed with a working `curl`, so it _should_ work out of the box.

Reply:
We also use `date`, `xxd`, `tr`, `sed`, `base64` and `cygpath`, but that should work.